<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>389</width>
    <height>309</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>GPIO Pin GUI</string>
  </property>
  <widget class="QPushButton" name="openDeviceButton">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>10</y>
     <width>75</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Open Device</string>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBoxEventCfg">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>210</y>
     <width>371</width>
     <height>91</height>
    </rect>
   </property>
   <property name="title">
    <string>Event Config:</string>
   </property>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>346</width>
      <height>58</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="eventTypeLabel">
       <property name="text">
        <string>eventType:</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="eventType">
       <item>
        <property name="text">
         <string>None</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Change</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Level 1</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Level 0</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Always</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QPushButton" name="setEventCfg">
       <property name="text">
        <string>Set</string>
       </property>
      </widget>
     </item>
     <item row="0" column="3">
      <widget class="QPushButton" name="getEventCfg">
       <property name="text">
        <string>Get</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="eventPeriodLabel">
       <property name="text">
        <string>eventPeriod:</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="eventPeriod"/>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>168</width>
     <height>22</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_2">
    <item row="0" column="0">
     <widget class="QLabel" name="pinLabel">
      <property name="text">
       <string>Pin:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QComboBox" name="pin"/>
    </item>
    <item row="0" column="2">
     <widget class="QCheckBox" name="isEnabled">
      <property name="text">
       <string>Enabled</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>41</y>
     <width>351</width>
     <height>159</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout_3">
    <item row="0" column="0">
     <widget class="QLabel" name="valueLabel">
      <property name="text">
       <string>value:</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="value">
      <property name="text">
       <string>-</string>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="getValueButton">
      <property name="text">
       <string>Get Value</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QLabel" name="outputValueLabel">
      <property name="text">
       <string>outputValue:</string>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QComboBox" name="outputValue">
      <item>
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="setOutputValue">
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
    </item>
    <item row="1" column="3">
     <widget class="QPushButton" name="getOutputValue">
      <property name="text">
       <string>Get</string>
      </property>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QLabel" name="directionLabel">
      <property name="text">
       <string>Direction:</string>
      </property>
     </widget>
    </item>
    <item row="2" column="1">
     <widget class="QComboBox" name="direction">
      <item>
       <property name="text">
        <string>0</string>
       </property>
      </item>
      <item>
       <property name="text">
        <string>1</string>
       </property>
      </item>
     </widget>
    </item>
    <item row="2" column="2">
     <widget class="QPushButton" name="setDirection">
      <property name="text">
       <string>Set</string>
      </property>
     </widget>
    </item>
    <item row="2" column="3">
     <widget class="QPushButton" name="getDirection">
      <property name="text">
       <string>Get</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QLabel" name="isOpenDrainLabel">
      <property name="text">
       <string>isOpenDrain:</string>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QCheckBox" name="isOpenDrain">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QLabel" name="isPullUpEnabledLabel">
      <property name="text">
       <string>isPullUpEnabled:</string>
      </property>
     </widget>
    </item>
    <item row="4" column="1">
     <widget class="QCheckBox" name="isPullUpEnabled">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="5" column="0">
     <widget class="QLabel" name="isPullDownEnabledLabel">
      <property name="text">
       <string>isPullDownEnabled:</string>
      </property>
     </widget>
    </item>
    <item row="5" column="1">
     <widget class="QCheckBox" name="isPullDownEnabled">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="6" column="0">
     <widget class="QLabel" name="isDebounceEnabledLabel">
      <property name="text">
       <string>isDebounceEnabled:</string>
      </property>
     </widget>
    </item>
    <item row="6" column="1">
     <widget class="QCheckBox" name="isDebounceEnabled">
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
